src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAOSpec.scala

```diff
@@ -38,6 +39,63 @@ class GraphWorkspaceDAOSpec extends FlatSpec with Matchers with OrientDbTestFixt
     }
   }
 
+  it should "save a new workspace with a realm" in withDefaultTestDatabase { dataSource =>
+    dataSource.inTransaction() { txn =>
+      val user = RawlsUser(RawlsUserSubjectId("user@secret.com"), RawlsUserEmail("user@secret.com"))
+      authDAO.saveUser(user, txn)
+      val secretGroup = makeRawlsGroup("SecretRealm", Set(user), Set.empty)
+      authDAO.saveGroup(secretGroup, txn)
+      val secretGroupRef = RawlsGroupRef(groupName = secretGroup.groupName)
+      val secretWorkspace = Workspace(
+        namespace = testData.wsName.namespace,
+        name = testData.wsName.name,
+        realm = Some(secretGroupRef),
+        workspaceId = "aWorkspaceId",
+        bucketName = "aBucket",
+        createdDate = testDate,
+        lastModified = testDate,
+        createdBy = "Secret User",
+        attributes = Map("workspace_attrib" -> AttributeString("foo")),
+        accessLevels = Map.empty
+      )
+      workspaceDAO.save(secretWorkspace, txn)
+      val savedWorkspace = workspaceDAO.findById(secretWorkspace.workspaceId, txn).get.workspace
+      assert {
```

Comment:
@jmthibault79 - Yeah, that fails. Here is the stack in case you are interested. Notice that the last modified date is milliseconds different.

```
[info] - should save a new workspace with a realm *** FAILED ***
[info]   Expected Workspace(myNamespace,myWorkspace,Some(RawlsGroupRef(RawlsGroupName(SecretRealm))),aWorkspaceId,aBucket,2016-02-22T17:55:25.354-05:00,2016-02-22T17:55:25.354-05:00,Secret User,Map(workspace_attrib -> AttributeString(foo)),Map(),false), but got Workspace(myNamespace,myWorkspace,Some(RawlsGroupRef(RawlsGroupName(SecretRealm))),aWorkspaceId,aBucket,2016-02-22T17:55:25.354-05:00,2016-02-22T17:55:28.796-05:00,Secret User,Map(workspace_attrib -> AttributeString(foo)),Map(),false) (GraphWorkspaceDAOSpec.scala:63)
```